server/task-json/src/test/java/org/apache/james/server/task/json/TaskDeserializerTest.java

```diff
@@ -0,0 +1,88 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ * http://www.apache.org/licenses/LICENSE-2.0                   *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ * ***************************************************************/
+package org.apache.james.server.task.json;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.io.IOException;
+
+import org.apache.james.task.Task;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import com.google.common.collect.ImmutableMap;
+
+class TaskDeserializerTest {
+
+    static final String TASK_AS_STRING = "{" +
+        "\"type\": \"testTask\"," +
+        "\"parameters\": {\"parameter\": \"1\"}" +
+        "}";
+
+    static final String UNREGISTERED_TASK_AS_STRING = "{" +
+        "\"type\": \"unknown\"," +
+        "\"parameters\": {\"parameter\": \"1\"}" +
+        "}";
+
+    static final String TWO_TYPES_TASK_AS_STRING = "{" +
+        "\"type\": \"testTask\"," +
+        "\"type\": \"unknown\"," +
+        "\"parameters\": {\"parameter\": \"1\"}" +
+        "}";
+
+    static final String MISSING_TASK_AS_STRING = "{" +
```

Comment:
why not putting those variables private?

Reply:
junit5 makes use of default visibility as a "less noisy" code style (because nobody would create dependencies between test classes anyway)